script/tool/lib/src/create_all_plugins_app_command.dart

```diff
@@ -259,4 +268,75 @@ dev_dependencies:${_pubspecMapString(pubspec.devDependencies)}
 
     return buffer.toString();
   }
+
+  Future<int> _genNativeBuildFiles() async {
+    // Only run on macOS.
+    // Other platforms don't need generation of additional files.
+    if (!io.Platform.isMacOS) {
+      return 0;
+    }
+
+    final io.ProcessResult result = io.Process.runSync(
+      flutterCommand,
+      <String>[
+        'pub',
+        'get',
+      ],
+      workingDirectory: _appDirectory.path,
+    );
+
+    print(result.stdout);
+    print(result.stderr);
+    return result.exitCode;
+  }
+
+  Future<void> _updateMacosPodfile() async {
+    // Only change the macOS deployment target if the host platform is macOS.
```

Comment:
This can be unconditional, since the project is created for all platform regardless.